protocol-definitions/DynamicConfig.yaml

```diff
@@ -1292,3 +1292,37 @@ methods:
           doc: |
             Properties of the data connection configuration.
     response: {}
+  - id: 18
+    name: addWanReplicationConfig
+    since: 2.7
+    doc: |
+      Adds a WAN replication configuration.
+    request:
+      retryable: false
```

Comment:
is this operation idempotent? if I call 2 times `ClientDynamicClusterConfig#addWanReplicationConfig` with the same config and name will it change the resulting config?

Reply:
Haven't addressed this in latest commit. Will do later.

Reply:
@ihsandemir this looks like it could be idempotent as it's predicated on the uniqueness of the configuration name, however it seems like others that follow a similar strategy have been marked as non-retriable. So, I think it's safer  just to follow the same pattern to be on safe side.